HW2/MultirankPalindromeTest.py

```diff
@@ -0,0 +1,13 @@
+num = int(input('Enter number:'))
+test = num
+dopel = 0
+while 1:
+    dopel += num % 10
+    num //= 10
+    if num / 10 <= 0:
```

Comment:
it can't be less than zero that's why it seems we can just write
```suggestion
if not num:
```